interpreters/lua/Kconfig

```diff
@@ -0,0 +1,90 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig INTERPRETERS_LUA
+	bool "Lua interpreter support"
+	default n
+	select ARCH_SETJMP_H
+	select LIBC_FLOATINGPOINT
+	select LIBC_LOCALE
+	select LIBM
+	select SYSTEM_SYSTEM
```

Comment:
why need system

Reply:
For the `os.execute()` method in Lua. For example:
```lua
    -- run sleep command from NSH
    os.execute("sleep 1")
```

Reply:
I assume these type of function expose through Lua module, it may better to implement the wrapper function like this:
```
lua_execute(...)
{
#ifdef CONFIG_SYSTEM_SYTEM
  system(...);
#else
  report error
#endif
}
```
so, if user don't need pay out the cost if they never use it. This trick is especially useful to some feature which increase a mount of code/data size.

Reply:
I wanted to avoid applying a patch to the Lua source, since it changes from release to release. Since the `system()` call is only used in `loslib.c`, which is only built if `CONFIG_INTERPRETER_LUA_CORE_LIBS` is enabled, how about having `INTERPRETER_LUA_CORE_LIBS` select `SYSTEM_SYSTEM`?

Reply:
Ok, it look fine.